src/components/ReportActionItem/MoneyRequestReportPreview/MoneyRequestReportPreviewContent.tsx

```diff
@@ -476,8 +476,8 @@ function MoneyRequestReportPreviewContent({
                             <View style={[reportPreviewStyles.wrapperStyle]}>
                                 <View style={[reportPreviewStyles.contentContainerStyle]}>
                                     <View style={[styles.expenseAndReportPreviewTextContainer, styles.overflowHidden]}>
-                                        <View style={[styles.flexRow, styles.justifyContentBetween, styles.gap3]}>
-                                            <View style={[styles.flexRow, styles.mw100, styles.flexShrink1, (isApproved || iouSettled) && styles.mtn1]}>
+                                        <View style={[styles.flexRow, styles.justifyContentBetween, styles.gap3, StyleUtils.getMinimumHeight(28)]}>
```

Comment:
Is 28 just a random number that fits? Maybe we should define a const for this value?

Reply:
28 is the height of the carousel buttons that show in this space on desktop, so the idea is that this area always has a min-height of 28px so we don't get any jumping or differences in layout depending on whether or not there are carousel buttons.

Reply:
makes sense, I think we could create a const and give a meaningful name @bernhardoj

Reply:
There is a `variables.h28`. Can I use that instead? (for both min height and height of the button)

Reply:
yeah if we have something that defines 28 then sure lets use it